google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugProcess.java

```diff
@@ -259,6 +259,21 @@ public void run() {
   }
 
   /**
+   * Clears out frames and variables
+   * Useful for breakpoint delete operations which result in no selected row
+   */
+  public void clearExecutionStack() {
+    SwingUtilities.invokeLater(new Runnable() {
+      @Override
+      public void run() {
+        if (!getXDebugSession().isStopped()) {
+          getXDebugSession().positionReached(new XSuspendContext() {});
```

Comment:
I don't really understand what's going on here.  How does this clear the stack?  Is this what Jetbrains does when you stop a debug session and it clears the debugger context?